src/Snapshot/SnapshotRepository.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the broadway/broadway-snapshotting package.
+ *
+ * (c) Qandidate.com <opensource@qandidate.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Broadway\Snapshotting\Snapshot;
+
+interface SnapshotRepository
+{
+    /**
+     * @param mixed $id
```

Comment:
yeah, I didn't want snapshotting to be blocked by this important but unresolved discussion: https://github.com/broadway/broadway/issues/120

added a note for now.